llpc/context/llpcCompiler.cpp

```diff
@@ -64,6 +64,7 @@
 #include <mutex>
 #include <set>
 #include <unordered_set>
+#include <vkgcDefs.h>
```

Comment:
nit: this is not a system header; should be `"vkgcDefs.h"`

Reply:
Actually this header is not needed, so I just removed it.  Thanks.